embiggen/visualizations/graph_visualizations.py

```diff
@@ -442,7 +444,7 @@ def plot_edge_types(
 
         self._set_legend(
             axes,
-            labels,
+            egde_labels,
             scatter.legend_elements()[0]
         )
         self._clear_axes(axes, "Edge types")
```

Comment:
Spelling
-- Return given arrats shuffled synchronously. (arrays)
--"""Reset the axes ticks and set the given title.""" => ``the axis ticks``
-- in _set_legend, "handles" should be "handlers" (param name and documentation), handle has a different meaning from handler
-- Documentation of function plot_node_degrees ("Plot common node types of provided graph.") is probably copy paste error?
-- line 426, egde_labels => edge_labels

Reply:
1. Fixed!
2. Within matplotlib the term `axes` is used to reference this type of object, is it a mistake in this context?
3. The term `handles` is the one used within matplotlib.
4. Fixed typo.
5. Fixed typo.

Reply:
axis=singular, axes plural, and so normally in English you would say axis ticks. matplotlib uses 'axes' (correctly) in other contexts, e.g., https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.set_xticks.html